etc/kayobe/kolla/config/ironic/ironic-conductor.conf

```diff
@@ -2,3 +2,11 @@
 # Enable automated cleaning. To save time this uses ATA secure erase when
 # supported.
 automated_clean=true
+
+[deploy]
+# Effectively disable shred.
+shred_random_overwrite_iterations = 0
```

Comment:
Does this mean we're not doing any cleaning of the disk?

Reply:
We can check back on this later, I'll merge now because this is running on the system.